apps/fxc-front/src/app/logic/track.ts

```diff
@@ -1,9 +1,21 @@
-import { extractGroupId } from '@flyxc/common';
+import {extractGroupId, RuntimeTrack} from '@flyxc/common';
 
 import { unwrapResult } from '@reduxjs/toolkit';
 
-import { store } from '../redux/store';
-import { fetchTrack } from '../redux/track-slice';
+import {AppDispatch, store} from '../redux/store';
+import { currentLeague } from '../redux/selectors';
+import {fetchTrack, RuntimeTrackId, setScore as setTrackStore} from '../redux/track-slice';
+// @ts-ignore
+import ScoreWorker from '../workers/score-track?worker';
```

Comment:
What is the byte size of that (how much does it add to the output) ?

We should consider loading it on demand as:
- not all sessions will use this,
- optimizing probably takes a few seconds, we can afford adding some more time for loading.

Reply:
The solver is now loaded lazilly